docs/release-notes/mia-platform-v12.mdx

```diff
@@ -11,3 +11,94 @@ import bannerImageUrl from './img/overview/v11-banner.png'
 :::info
 This page is work in progress, come back soon for updates!
 :::
+
+Welcome to Mia-Platform v12! This latest major release is focused on scaling Developer Experience (DevX).
+Let's take a closer look to all the new features included in Mia-Platform v12.
+
+## GitOps in Pull
+
+Mia-Platform Consle used to support just GitOps in Push approach. 
+Now, we implemented also the GitOps in Pull approach, letting you decide which one to choose.
+When starting a new Project, you will be able to choose which approach adopt for that project.
+The GitOps in Pull approach has several benefits from a security perspective.
+To learn more about this feature, take a look at the video below or read the [full documentation](#).
+
+## Mia-Platform Fast Data
+
+We further improved the No-Code functionalities of Mia-Platform Fast Data, and now you it will only takes you 1 hour to aggregate a Single View from scratch.
+Below you can find a video that shows you how it works. To learn more, please refer to the [documentation](#).
+
+### Import and export between projects
+
+Moreover, now you can import and export several assets from one project to another.
+The assets you can import and export include:
+
+- Projections;
+- Single Views;
+- Configurations.
+
+Everything is fully explained in the related [documentation](#).
+
+### Refactored architecture
+
+We refactored the architecture of Mia-Platform Fast Data to empower new technology integration.
+Now, it will be easier to integrate with new services.
+Also, the new architecture improves the overall performance.
+Learn more on the [documentation](#).
+
+## Marketplace
+
+Now you can configure individual services on the Marketplace to be accessible exclusively by your company's members.
+Watch the video below to learn more about how it works and refer to the [documentation](#) for further details.
+
```

Comment:
<iframe width="560" height="315" src="https://www.youtube.com/embed/YFwTAZmPSsc?si=KQz6LrorFth0GIlo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>